### PR TITLE
Make adding event separator configurable through annotation element

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/io/file/FileSink.java [hunk 1]
```diff
@@ -76,6 +76,16 @@
                         type = {DataType.BOOL},
                         optional = true,
                         defaultValue = "true"
+                ),
+                @Parameter(
+                        name = "add.event.separator",
```
Comment (suhothayan) [inline component/src/main/java/org/wso2/extension/siddhi/io/file/FileSink.java]: It should be line separator

**Review by suhothayan — CHANGES_REQUESTED** [review]

File: component/src/main/java/org/wso2/extension/siddhi/io/file/FileSink.java (hunk 1 above)
Comment (niveathika) [reply component/src/main/java/org/wso2/extension/siddhi/io/file/FileSink.java]: Fixed in 92b2f53

File: component/src/main/java/org/wso2/extension/siddhi/io/file/FileSink.java [hunk 2]
```diff
@@ -128,6 +139,11 @@ protected void init(StreamDefinition streamDefinition, OptionHolder optionHolder
         if (Constants.TRUE.equalsIgnoreCase(append)) {
             properties.put(Constants.APPEND, append);
         }
+        String mapType = streamDefinition.getAnnotations().get(0).getAnnotations().get(0).getElements().get(0)
+                .getValue();
+        addEventSeparator = !mapType.equalsIgnoreCase("csv") &&
+                Boolean.parseBoolean(optionHolder.validateAndGetStaticValue(Constants.ADD_EVENT_SEPARATOR,
```
Comment (suhothayan) [inline component/src/main/java/org/wso2/extension/siddhi/io/file/FileSink.java]: Check if ADD_EVENT_SEPARATOR exist? 
If it exists use that value. 
only if it does not exist use the defaults (true / in csv case use false)

Comment (niveathika) [reply component/src/main/java/org/wso2/extension/siddhi/io/file/FileSink.java]: Fixed in 2eb346c

**Review by suhothayan — APPROVED** [review]

**Pull request merged**